qa/acceptance-tests/src/test/java/io/camunda/it/client/MappingsByGroupSearchTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.client;
+
+import static io.camunda.qa.util.multidb.CamundaMultiDBExtension.TIMEOUT_DATA_AVAILABILITY;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.client.CamundaClient;
+import io.camunda.client.api.search.response.Mapping;
+import io.camunda.qa.util.multidb.MultiDbTest;
+import io.camunda.zeebe.test.util.Strings;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@MultiDbTest
+public class MappingsByGroupSearchTest {
+
+  private static CamundaClient camundaClient;
+
+  private static final String MAPPING_ID_1 = Strings.newRandomValidUsername();
+  private static final String MAPPING_ID_2 = Strings.newRandomValidUsername();
+  private static final String MAPPING_ID_3 = Strings.newRandomValidUsername();
+  private static final String GROUP_ID = Strings.newRandomValidIdentityId();
+
+  @BeforeAll
+  static void setup() {
+    createMapping(MAPPING_ID_1);
+    createMapping(MAPPING_ID_2);
+    createMapping(MAPPING_ID_3);
+
+    createGroup(GROUP_ID);
+
+    assignMappingToGroup(MAPPING_ID_1, GROUP_ID);
+    assignMappingToGroup(MAPPING_ID_2, GROUP_ID);
+
+    waitForGroupsToBeUpdated();
+  }
+
+  @Test
+  void shouldReturnMappingsByGroup() {
+    final var mappings = camundaClient.newMappingsByGroupSearchRequest(GROUP_ID).send().join();
+
+    assertThat(mappings.items().size()).isEqualTo(2);
+    assertThat(mappings.items())
+        .extracting(Mapping::getMappingId)
+        .contains(MAPPING_ID_1, MAPPING_ID_2);
+  }
+
+  @Test
+  void shouldReturnMappingsByGroupFiltered() {
```

Comment:
❓ Should we add a case for sorting as well? Or is this covered elsewhere already?

Reply:
Yeah, I can add a test case